Experimenter/Experimenter.UI/ApplicationMain.cs

```diff
@@ -6,13 +6,15 @@
 
 namespace DistributedExperimentation.Experimenter.UI
 {
-    class ApplicationMain
+    public class ApplicationMain
     {
-        static void Main(string[] args)
+        public static void Main(string[] args)
         {
             try {
                 Dictionary<string, string> parsedArgs = parseArguments(args);
-                String pluginPath = getPluginFilePath("ExecutorPlugin", Directory.GetCurrentDirectory());
+                if (!parsedArgs.ContainsKey("--pluginPath"))

```

Comment:
I had to add an plugin path to not seek through the whole project structure for my unit test.
Otherwhise i had to compile the plugin and copy it into Experimenter.UI\bin\Debug\netcoreapp2.2 and that seems wrong to me.

Reply:
[See my answer](https://github.com/MrMong/DistributedDockerExperimentation/pull/1#discussion_r295133526)